protos/streams/v1/streams.pb.go

```diff
@@ -23,15 +23,16 @@ package v1
 
 import (
 	context "context"
+	reflect "reflect"
```

Comment:
Why did these change? How did you generate the protos?

Reply:
This is me running `goimports -local github.com/docker/compose-cli -w .` after `make protos`. I'll remove this to not change all generated files in this PR, only volume changes required